src/pluginops/pluginops.py

```diff
@@ -576,12 +568,11 @@ def configure(virtual_source, snapshot, repository):
     }
     configure_script = pkgutil.get_data('resources', 'provision.sh')
     result = libs.run_bash(virtual_source.connection, configure_script,environment_vars,check=False)
-    logger.debug(result)
     output = result.stdout.strip()
     std_err=result.stderr.strip()
     exit_code = result.exit_code
     if exit_code == 0:
-        logger.debug("Pre-Snapshot/Restore_DB successful "+output)
+        logger.debug("Pre-Snapshot/Restore_DB successful"+output)
```

Comment:
Not sure if the intention here was to remove the output. Please check.

Reply:
The output contains password as well , so had to remove this line